### PR TITLE
Remove unused code from cadence.internal package

### PR DESCRIPTION
**Pull request by natemort**
**What changed?**
Remove unused code from internal packages to improve code coverage and reduce maintenance burden. Users shouldn't be accessing internals.

**Why?**
Improving code coverage and reducing maintenance burden

**How did you test it?**
Unit tests

**Potential risks**
User code may be accessing internals within the client and could break. Generally this should be a compile time breakage and we do not support accessing internals.

**Release notes**

**Documentation Changes**

### CODE REVIEW

**Review by 3vilhamster — APPROVED** [review]

**Review by 3vilhamster — COMMENTED** [review]
I'm unsure if we want to delete this, but I could not find usages of this in monorepos or in open sourced usages.

File: src/main/java/com/uber/cadence/internal/common/WorkflowExecutionUtils.java [hunk 1]
```diff
@@ -621,26 +437,6 @@ public static GetWorkflowExecutionHistoryResponse getHistoryPage(
     return history;
   }
 
-  /** Returns workflow instance history in a human readable format. */
-  public static String prettyPrintHistory(
```
Comment (3vilhamster) [inline src/main/java/com/uber/cadence/internal/common/WorkflowExecutionUtils.java:625]: This could be useful; I wonder why have you decided to drop this.

File: src/main/java/com/uber/cadence/internal/sync/WorkflowInternal.java [hunk 2]
```diff
@@ -228,26 +220,6 @@ public static <T> T newContinueAsNewStub(
...
-  /**
-   * Execute activity by name.
-   *
-   * @param name name of the activity
-   * @param resultClass activity return type
-   * @param args list of activity arguments
-   * @param <R> activity return type
-   * @return activity result
-   */
-  public static <R> R executeActivity(
```
Comment (3vilhamster) [inline src/main/java/com/uber/cadence/internal/sync/WorkflowInternal.java:240]: I wonder why is this removed?

**Review by Shaddoll — APPROVED** [review]

File: src/main/java/com/uber/cadence/internal/common/WorkflowExecutionUtils.java (hunk 1 above)
Comment (natemort) [reply src/main/java/com/uber/cadence/internal/common/WorkflowExecutionUtils.java:625]: I think it's potentially useful but it's long unused and untested. If we need similar functionality we can rewrite it.

File: src/main/java/com/uber/cadence/internal/sync/WorkflowInternal.java (hunk 2 above)
Comment (natemort) [reply src/main/java/com/uber/cadence/internal/sync/WorkflowInternal.java:240]: I'm not really sure why this exists, it's long unused and untested. I think it's best we just remove it.
